index.html

```diff
@@ -2085,6 +2095,36 @@ <h3>publicKeyHex</h3>
         </pre>
       </section>
 
+      <section id="publicKeyMultibase" about="https://w3id.org/security#publicKeyMultibase"
+      typeof="rdf:Property">
+        <h3>publicKeyMultibase</h3>
+        <p>
+The public key multibase property is used to specify the multibase-encoded
+version of a public key.
```

Comment:
Suggestion choice without the trailing advice:

```suggestion
version of a public key. Each key type definition is expected to specify
one or more required encoding bases.
```